kpi/models/asset_version.py

```diff
@@ -74,3 +74,20 @@ def __str__(self):
             self.asset.uid, self.uid,
             self.date_modified.strftime('%Y-%m-%d %H:%M'),
             ' (deployed)' if self.deployed else '')
+
+    def save(
+        self,
+        force_insert=False,
```

Comment:
This would be better as *args and **kwargs. That way if django ever changed it, this code wouldn't break.

Reply:
Sure! 
But many places in the code (because of me 😇 ) are using the current signature and not *args, **kwargs. Should we create an issue to change them or should we wait until it break - if it breaks one day?

Reply:
For new/modified code - I would advise using *kwargs, unless upstream documentation says otherwise. If I was using a variable such as force_insert, I would also declare that explicitly instead of fetching from kwargs.

IMO It's not a valuable time investment going back and changing this. Yes - if it breaks, we can change it.